src/components/Incidents/IncidentsChart/IncidentsChart.jsx

```diff
@@ -6,103 +6,147 @@ import {
   ChartBar,
   ChartGroup,
   ChartLabel,
-  ChartThemeColor,
   ChartTooltip,
   ChartVoronoiContainer,
 } from '@patternfly/react-charts';
-import { VictoryZoomContainer } from 'victory-zoom-container';
 import { Bullseye, Card, CardTitle, Spinner } from '@patternfly/react-core';
 import global_danger_color_100 from '@patternfly/react-tokens/dist/esm/global_danger_color_100';
 import global_info_color_100 from '@patternfly/react-tokens/dist/esm/global_info_color_100';
 import global_warning_color_100 from '@patternfly/react-tokens/dist/esm/global_warning_color_100';
-import { createIncidentsChartBars, createDateArray, formatDate, generateDateArray } from '../utils';
+import { createIncidentsChartBars, formatDate, generateDateArray } from '../utils';
+import { getResizeObserver } from '@patternfly/react-core';
 
-const IncidentsChart = ({ incidentsData, chartDays }) => {
+const IncidentsChart = ({ incidentsData, chartDays, chooseIncident, tableLoaded }) => {
   const [isLoading, setIsLoading] = React.useState(true);
   const [chartData, setChartData] = React.useState();
-  const dateValues = generateDateArray(chartDays);
+  const [width, setWidth] = React.useState(0);
+  const containerRef = React.useRef(null);
+  const [hiddenSeries, setHiddenSeries] = React.useState(null);
+  const handleResize = () => {
+    if (containerRef.current && containerRef.current.clientWidth) {
+      setWidth(containerRef.current.clientWidth);
+    }
+  };
+  React.useEffect(() => {
+    const observer = getResizeObserver(containerRef.current, handleResize);
+    handleResize();
+    return () => observer();
+  }, []);
   React.useEffect(() => {
     setIsLoading(false);
     setChartData(incidentsData.map((incident) => createIncidentsChartBars(incident)));
   }, [incidentsData]);
+  const dateValues = generateDateArray(chartDays);
+
+  const isHidden = (group_id) => hiddenSeries !== null && hiddenSeries !== group_id;
+  const clickHandler = (data, datum) => {
+    tableLoaded(true);
```

Comment:
not sure why this needs to change here. This is probably changing the state of the parent component forcing a re-render of this component.